code/go/0chain.net/blobbercore/datastore/store.go

```diff
@@ -60,7 +60,7 @@ func (store *Store) Close() {
 
 func (store *Store) CreateTransaction(ctx context.Context) context.Context {
 	db := store.db.Begin()
-	return context.WithValue(ctx, CONNECTION_CONTEXT_KEY, db)
+	return context.WithValue(ctx, CONNECTION_CONTEXT_KEY, db) //nolint:staticcheck // changing type might require further refactor
```

Comment:
not sure if it's necessary to create a `type ConnectionContextKey string` for this, but `context.WIthValue` itself warns that key should be of built-in type.